lib/common/db-renderable-content.js

```diff
@@ -118,6 +118,11 @@ function dbRenderableContentServiceFactory(
 
     DbRenderableContentService.prototype.render = function render (name, options, scope) {
         return this.get(name, scope).then(function (template) {
+            if (!template) {
+                throw new Errors.TemplateRenderError(
```

Comment:
@heckj I was hoping no one would notice that :)  I wasn't quite quick enough pushing the fix.